test/esjs.spec.js

```diff
@@ -276,19 +276,38 @@ describe('.search()', () => {
   });
```

Comment:
nope. I should add that once we decide how it should work in the other comment chain